projects/ngx-cookie-consent/src/lib/config/translated-text.type.ts

```diff
@@ -0,0 +1,4 @@
+export type TranslatedText = {
+    [language: string]: string;
+    default: string;
```

Comment:
default is not required here - imho - the user should define all enabled languages, at least one language